charts/zenko/values.yaml

```diff
@@ -88,3 +88,10 @@ redis:
     enabled: false
   metrics:
     enabled: true
+
+redis-ha:
```

Comment:
Can add an ```enabled: true``` that can be referenced in the requirements.yaml

Reply:
Same here.